xeth/whisper.go

```diff
@@ -36,7 +36,7 @@ func (self *Whisper) Post(payload string, to, from string, topics []string, prio
 			TTL:    time.Duration(ttl) * time.Second,
 			To:     crypto.ToECDSAPub(common.FromHex(to)),
 			From:   key,
-			Topics: whisper.TopicsFromString(topics...),
+			Topics: whisper.NewTopicsFromStrings(topics...),
```

Comment:
As I see it, the stdlib convention is **New**Something**String**

 * NewBufferString http://golang.org/pkg/bytes/#NewBufferString
 * NewReaderSize http://golang.org/pkg/bufio/#NewReaderSize

So I think we should either do
 * TopicsFromStrings as previously to drop the new and don't conflict with the above style
 * NewTopicsStrings to adhere completely, but this is a bit... weird.

I think I prefer the TopicsFromStrings approach.